pygmt/datasets/load_remote_dataset.py

```diff
@@ -226,6 +226,29 @@ class GMTRemoteDataset(NamedTuple):
             "03m": Resolution("03m", registrations=["gridline"], tiled=True),
         },
     ),
+    "moon_relief": GMTRemoteDataset(
+        title="Moon relief",
+        name="moon relief",
+        long_name="Moon relief",
```

Comment:
For the datasets of the Earth, underscores are used for `name`.
```suggestion
        title="Moon relief",
        name="moon_relief",
        long_name="Moon relief",
```